engine/install/debian.md

```diff
@@ -119,18 +119,6 @@ Raspbian.
    $ sudo apt-get update
    ```
 
-   > Receiving a GPG error when running `apt-get update`?
-   >
-   > Your default [umask](https://en.wikipedia.org/wiki/Umask){: target="blank"
-   > rel="noopener" } may be incorrectly configured, preventing detection of the
-   > repository public key file. Try granting read permission for the Docker
-   > public key file before updating the package index:
-   >
-   > ```console
-   > $ sudo chmod a+r /etc/apt/keyrings/docker.gpg
-   > $ sudo apt-get update
-   > ```
```

Comment:
🎉 

before we fully celebrate, let's also get some more eyes on this in case we're overlooking scenarios where someone could still run into this.

Overall I _think_ we should be fine (assuming users just ran the steps above), and I _guess_ if someone ran those steps with the old variant, they would already have run into the issue.

Reply:
Oh! 🤦 I think there _is_ still a situation; we now fixed the directory permissions, but we don't explicitly set read permissions on the `docker.gpg` itself.

I guess we could either keep a note (but I hate "too many notes" as they distract from the overall flow), or we could add a line to always set the permissions;

```bash
$ curl -fsSL {{ download-url-base }}/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
$ sudo chmod a+r /etc/apt/keyrings/docker.gpg
```


I double-checked and yes, looks like we still may have to (here's with a `0066` umask);

```bash
umask 0066
curl -fsSL https://download.docker.com/linux/ubuntu/gpg | gpg --dearmor -o /etc/apt/keyrings/docker2.gpg


ls -l /etc/apt/keyrings/
total 8
-rw-r--r-- 1 root root 2760 Jun  7  2022 docker.gpg
-rw------- 1 root root 2760 Apr  8 16:20 docker2.gpg

umask 0022
```

Reply:
added `chmod` to the previous step - I agree that too many notes creates clutter and makes it hard to read